tests/browscap/build

```diff
@@ -40,15 +54,15 @@ foreach (new FilesystemIterator(__DIR__ . '/vendor/browscap/browscap/tests/issue
                     'version' => $data['properties']['Version'],
                 ],
                 'platform' => [
-                    'name'    => isset($data['properties']['Platform']) ? $data['properties']['Platform'] : 'unknown',
+                    'name'    => $data['properties']['Platform'] ?? 'unknown',
                     'version' => $data['properties']['Platform_Version'],
                 ],
                 'device' => [
-                    'name'     => $data['properties']['Device_Name'],
-                    'brand'    => $data['properties']['Device_Maker'],
+                    'name'     => $data['properties']['Device_Code_Name'],
+                    'brand'    => $data['properties']['Device_Brand_Name'],
```

Comment:
You changed these back in the parsers, but forgot the tests :)

Reply:
reverted